pages/api/graphql/index.ts

```diff
@@ -1,10 +1,11 @@
 // import { error as LogError } from "services/logging";
-import {withCors} from "middleware";
 import {NextApiRequest, NextApiResponse} from "next";
 import {getToken} from "next-auth/jwt";
 import getConfig from "next/config";
 import {Octokit} from "octokit";
 
+import {withCors} from "middleware";
```

Comment:
{withCors} seems to have been added to some pages but not used -- doublecheck this